presto-main/src/main/java/io/prestosql/SessionRepresentation.java

```diff
@@ -61,6 +61,7 @@
     private final Map<CatalogName, Map<String, String>> catalogProperties;
     private final Map<String, Map<String, String>> unprocessedCatalogProperties;
     private final Map<String, SelectedRole> roles;
+    private final Set<String> groups;
```

Comment:
Let's move this to be directly after `user` everywhere in this class, to be consistent with the identity classes